src/Core/Domain/State/ValueObject/StateIdInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
```

Comment:
Missing license header

Reply:
Something that I didn't know at yet ! thx